test/extended/images/mongodb_replica.go

```diff
@@ -0,0 +1,58 @@
+package images
+
+import (
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/db"
+)
+
+var _ = g.Describe("[images][mongodb] openshift mongodb replication", func() {
+	defer g.GinkgoRecover()
+
+	const (
+		templatePath     = "https://raw.githubusercontent.com/openshift/mongodb/master/2.4/examples/replica/mongodb-clustered.json"
```

Comment:
What is the approach in other extended tests, do we fetch templates from master?! Can't we use a fixture?

Hmm.. if we eventually want to use this extended test to test PRs in the mongodb repository, this value cannot be hard-coded to master... I wonder how we do this for MySQL and PostgreSQL.